flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/NtTable.java

```diff
@@ -0,0 +1,47 @@
+package com.flash3388.frc.nt.combined;
```

Comment:
Can just be in `nt` package. I don't really understand the `combined` naming. It's also barely a thing.

Reply:
Yeah, my bad